packages/common-ts/src/base-service/base-service-v2.ts

```diff
@@ -386,9 +386,17 @@ export abstract class BaseServiceV2<
       const app = express()
 
       // Body parsing.
-      app.use(bodyParser.json())
       app.use(bodyParser.urlencoded({ extended: true }))
 
+      // Keep the raw body around in case the application needs it.
+      app.use(
+        bodyParser.json({
```

Comment:
This still will parse the json body as before?

Reply:
Yes - the parsing will continue as long as `verify` does not through an error.